contrib/ansible/cri-containerd.yaml

```diff
@@ -15,26 +15,25 @@
     
     - name: "Start CRI-Containerd"
       systemd: name=cri-containerd daemon_reload=yes state=started enabled=yes
-    
-    - name: "Set bridge-nf-call-iptables"
-      lineinfile:
-        line: "net/bridge/bridge-nf-call-iptables = 1"
-        dest: /etc/sysctl.conf
-        insertafter: 'EOF'
-        regexp: '\/net\/bridge\/bridge-nf-call-iptables = 1'
-        state: present
-      ignore_errors: true
-    
+
+    - sysctl:
```

Comment:
can u add name for the task

Reply:
Actually I wanted to do that, but don't know how to do it... Let me lookup the document.

Reply:
Done